src/content/1/en/part1d.md

```diff
@@ -1016,9 +1016,8 @@ The internet is full of React-related material. However, we use such a new style
 
 You may find the following links useful:
 
-- The React [official documentation](https://reactjs.org/docs/hello-world.html) is worth checking out at some point, although most of it will become relevant only later on in the course. Also, everything related to Class-components is irrelevant to us.
-- Beware the official React [tutorial](https://reactjs.org/tutorial/tutorial.html), it's not very good.
```

Comment:
imo there's no value in stating that this tutorial is not good without explaining why